### PR TITLE
Fixed some typos

### PR DESCRIPTION
**Pull request by kevin-schumann**
The title should be sufficient.

### CODE REVIEW

File: w01_big_picture/t04_rl_nutshell.tex [hunk 1]
```diff
@@ -133,7 +133,7 @@
 \begin{frame}[c]{Episode}
 	\begin{itemize}
 		\item An episode is sequence of state-action(-reward) pair (i.e., steps)
-		\item The end of an episode is called an horizon
+		\item The end of an episode is called a horizon
```
Comment (kevin-schumann) [inline w01_big_picture/t04_rl_nutshell.tex:136]: As you pronounce the h in "horizon" it is "a horizon".

File: w03_policy_evaluation/t05_td.tex [hunk 2]
```diff
@@ -39,7 +39,7 @@
 	\item Recall Bellman operator (if known MDP models)
 	$$B^\pi V(s) = r(s,\pi(s)) + \gamma \sum_{s'\in S} p(s'\mid s, \pi(s)) V(s')$$
 	\pause
-	\item Insight: have an estimate of $V^\pi$, use to estimate expected return
+	\item Insight: have an estimate of $V^\pi$, used to estimate expected return
```
Comment (TheEimer) [inline w03_policy_evaluation/t05_td.tex:42]: These are two correct grammatical forms that can be used, but the original is fine

File: w06_deep_rl/t05_dueling.tex [hunk 3]
```diff
@@ -60,7 +60,7 @@
 	%\item Do we expect that this architecture will result in us learning a good estimate of true $V$ or $A$?
 	\bigskip
 	\pause
-	\item Challenge: There hasn't have to be a unique advantage function
+	\item Challenge: There hasn't had to be a unique advantage function
```
Comment (TheEimer) [inline w06_deep_rl/t05_dueling.tex:63]: No, none of these are correct, 'doesn't have to be' is right here

File: w07_policy_search/t03_finite_difference.tex [hunk 4]
```diff
@@ -36,10 +36,10 @@
 	
 	\begin{itemize}
 		\item To evaluate policy gradient of $\pi_\theta(s,a)$
-		\item For each dimensions $k\in [1,n]$
+		\item For each dimension $k\in [1,n]$
 		\begin{itemize}
 			\item Estimate $k$-th partial derivative of objective function wrt $\theta$
-			\item By pertubating $\theta$ by small amount $\epsilon$ in $k$-th dimension
+			\item By perpetuating $\theta$ by small amount $\epsilon$ in $k$-th dimension
```
Comment (TheEimer) [inline w07_policy_search/t03_finite_difference.tex:42]: No, translate the original and your "correction", you'll see the difference

File: w07_policy_search/t06_pg_algorithms.tex [hunk 5]
```diff
@@ -94,11 +94,11 @@
 	
 	$$\nabla_\theta \mathbb{E}_\tau [R] = \mathbb{E}_\tau \left[ \sum_{t=0}^{T-1} \nabla_\theta \log \pi(a_t \mid s_t; \theta) \left( \sum_{t'=t}^{T-1} r_{t'} - b(s_t) \right) \right] $$
 	
-	\item For any choise of $b$, gradient estimator is unbiased
+	\item For any choice of $b$, gradient estimator is unbiased
 	\item Near optimal choice is the expected return
 	$$b(s_t) \approx \mathbb{E} [r_t + r_{t+1} +\ldots + t_{T-1}] $$
 	
-	\item Interpretation: increase logprob of action $a_t$ proportionally to how much returns $\sum_{t'=t}^{T-1} r_{t'}$ are better than expected
+	\item Interpretation: increase logprob of action $a_t$ proportionally to how many returns $\sum_{t'=t}^{T-1} r_{t'}$ are better than expected
```
Comment (TheEimer) [inline w07_policy_search/t06_pg_algorithms.tex:101]: I think we want much here, which measures the amount of reward the exceeds the expectation vs many which measures the number of steps in which the reward is better

File: w00_init/t01_big_picture.tex [hunk 6]
```diff
@@ -247,7 +247,7 @@
 		\end{itemize}
 		\item Basics of \alert{Machine Learning} (mandatory)
 		\begin{itemize}
-			\item Classification, regression, clustering, decision tree, training-test split, cross validation, pre-processing \ldots
+			\item Classification, regression, clustering, decision tree, training-test split, cross validation, preprocessing \ldots
```
Comment (TheEimer) [inline w00_init/t01_big_picture.tex:250]: I would say both works, I don't think this is a mistake (two other instances of this below I haven't commented on).

**Review by TheEimer — COMMENTED** [review]
Some things here are not corrections.

File: w00_init/t01_big_picture.tex [hunk 7]
```diff
@@ -247,7 +247,7 @@
 		\end{itemize}
 		\item Basics of \alert{Machine Learning} (mandatory)
 		\begin{itemize}
-			\item Classification, regression, clustering, decision tree, training-test split, cross validation, preprocessing \ldots
+			\item Classification, regression, clustering, decision tree, training-test split, cross validation, pre-processing \ldots
 			\item to catch up (if nec.): \url{https://www.coursera.org/learn/machine-learning}
```
Comment (kevin-schumann) [inline w00_init/t01_big_picture.tex]: You are right, preprocessing can be written that way. Writing "pre-processing" makes more sense when saying "pre- and post-processing" in my opinion.

**Review by mlindauer — APPROVED** [review]
Thanks for finding and fixing all these typos!

**Pull request merged**